meetings/tool/src/main/resources/create-meeting_eu.properties

```diff
@@ -0,0 +1,26 @@
+add_to_calendar=Add meeting to calendar
```

Comment:
@pablosanroman would you mind contributing a translation? Thanks!

Reply:
Hi @mpellicer 

all_participants_notification= Guztiak - Bidali jakinarazpena partaide guztiei
all_participants= Guneko partaide guztiak
cancelar= Utzi
close_date= Bukatzeko data
error_create_meeting_500= Bilera sortzean arazo bat egon da
error_updating_meeting_500= Akats bat egon da bilera eguneratzean
error_create_meeting_unknown= Akats ezezagun bat egon da bilera bat sortzean. Erantzun-kodea:
error_video_conferencing_config= Bileren tresnak, gutxienez, Microsoft Teams edo Zoom bezalako hornitzaile bat behar du. Kontaktatu behar duzu zure zerbitzu-administratzailea edo hornitzailea. 
group_participants= Hautatutako taldeetako partaideak
info_no_groups= Taldeen hautaketa desgaituta dago gune honek ez baitu talderik.
meeting_description= Deskribapena
meeting_title= Izenburua
ms_teams= Microsofteko taldeak
no_notification= Bat ere ez - Jakinarazpenik gabe
notificaciones= Jakinarazpenak
open_date= Hasteko data
participantes_selection= Partaideen hautaketa
guardar= Gorde
sección_disponibilidad= 3. Erabilgarritasuna
section_meeting_information= 1. Bilerari buruzko informazioa
sección_notificaciones= 4. Jakinarazpenak
section_participants= 2. Partaideak
select_groups= Hautatu taldeak
validación_close_date= Bukatzeko datak hasteko dataren ondorengoa izan behar du
video_conferencing_service= Bideokonferentzia zerbitzua